master/internal/user/postgres_users.go

```diff
@@ -117,6 +117,26 @@ func Update(
 	})
 }
 
+// SetActive changes multiple users' activation status.
+func SetActive(
+	ctx context.Context,
+	updateIDs []model.UserID,
+	activate bool,
+) error {
+	return db.Bun().RunInTx(ctx, &sql.TxOptions{}, func(ctx context.Context, tx bun.Tx) error {
```

Comment:
do we need this transaction for a single query?

Reply:
leftover from the more complex `patchUser`, will remove transaction part